unified-runtime/cmake/FetchLevelZero.cmake

```diff
@@ -3,146 +3,139 @@
 # See LICENSE.TXT
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-set(UR_LEVEL_ZERO_LOADER_LIBRARY "" CACHE FILEPATH "Path of the Level Zero Loader library")
-set(UR_LEVEL_ZERO_INCLUDE_DIR "" CACHE FILEPATH "Directory containing the Level Zero Headers")
-set(UR_LEVEL_ZERO_LOADER_REPO "" CACHE STRING "Github repo to get the Level Zero loader sources from")
-set(UR_LEVEL_ZERO_LOADER_TAG "" CACHE STRING " GIT tag of the Level Loader taken from github repo")
-set(UR_COMPUTE_RUNTIME_REPO "" CACHE STRING "Github repo to get the compute runtime sources from")
-set(UR_COMPUTE_RUNTIME_TAG "" CACHE STRING " GIT tag of the compute runtime taken from github repo")
-
-# If UR_COMPUTE_RUNTIME_FETCH_REPO is set to OFF, then UR_COMPUTE_RUNTIME_REPO should be defined and
-# should point to the compute runtime repo.
-set(UR_COMPUTE_RUNTIME_FETCH_REPO ON CACHE BOOL "Flag to indicate wheather to fetch the compute runtime repo")
-
-# Copy Level Zero loader/headers locally to the build to avoid leaking their path.
```

Comment:
I removed this because it made no sense to me, I don't understand why we would want to avoid 'leaking the path' of preinstalled system headers found and used by the build system. If someone has some idea on why we ever did this, let me know.

Reply:
I think this was to avoid leaking the `/nfs` path. eg:  The header is preinstalled in internal `/nfs` path, if we don't do this, the nfs path will be visible in the final binary, which might contains some sensitive info.

Reply:
Sorry, finally back from vacation. We don't do this for any other deps so it's unclear to my why L0 is special